hack/version.sh

```diff
@@ -90,13 +90,26 @@ function kubevirt::version::get_version_vars() {
             fi
         fi
     fi
+
+    commit_unix_timestamp="$(git show -s --format=format:%ct HEAD)"
+    KUBEVIRT_CONTAINER_TAG="$(compose_container_tag "${commit_unix_timestamp}" "${KUBEVIRT_GIT_COMMIT}")"
```

Comment:
I am not sure about this. It may make sense to simply leave the tag information empty (or maybe default it to something nonexistent) if `DOCKER_TAG` is not set.

Reply:
Initializing it here make sure that the the tag is "burned to the binary", this was the problem in the first place
there was no tag for images we use on tests thus it used the default (`latest`).
If we don't initialize it, you must export DOCKER_TAG with the correspond tag for the conformance tests to work
properly.
What you think?

Reply:
Yes, I think just exporting the DOCKER_TAG on the build commands should make it available to get burned in. I just want to avoid that we overthink things and I think it is easier to just go with one tag which specifies the version for all.